public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -113,10 +119,25 @@ export enum AlertingAction {
 const AlwaysSupported = true;
 const NotSupported = false;
 
-export type Action = AlertmanagerAction | AlertingAction | AlertRuleAction;
+export type Action = AlertmanagerAction | AlertingAction | AlertRuleAction | FolderAction;
 export type Ability = [actionSupported: boolean, actionAllowed: boolean];
 export type Abilities<T extends Action> = Record<T, Ability>;
 
+/**
+ * This one will check for folder abilities
+ */
+export const useFolderAbilities = (): Abilities<FolderAction> => {
```

Comment:
I do wonder if we should use `folderAbilities` name here. It conflicts a bit with folder abilities like renaming, deleting and it might be confusing.
How about having `useBulkActionAbilities` instead?

Reply:
I agree, but I would use something including the level of these bulk actions (folder): `useFolderBulkActionAbilities`